src/ui/molecules/bridge.burn/bridge.burn.amount.js

```diff
@@ -34,6 +37,9 @@ export const BridgeBurnInput = ({
   return (
     <>
       <div className="w-fit self-center px-0 py-0 scale-[0.8] md:scale-[1] z-10">
+        <div className=" xl:mr-5 tracking-wider pr-2 w-full flex justify-end text-[10px] text-badger-yellow-neon-400">
+          <span>Your Balance: {balances[token].toFixed(6) + " " + token}</span>
```

Comment:
You should validate that balances has a value for the `token` key, something like 

`(balances[token] ?? 0).toFixed(6)`

Reply:
Even better would be putting in a function `getBalance(token)` which would allow for a unit test to be written

Reply:
I updated the PR to address this.